src/Config/Loader/PhpFileLoader.php

```diff
@@ -52,7 +52,29 @@ public function load($file, $type = null)
             $code = substr($code, 0, -2);
         }
 
-        return rtrim($code)."\n";
+        // declare() statements
+        $lines = explode("\n", $code);
+        $codeNew = '';
+        $processed = false;
+
+        foreach ($lines as $line) {
+            if ('' === $line || $processed) {
+                $codeNew .= $line . "\n";
+                continue;
+            }
+
+            // Until now, every line was empty which means this is the first
+            // one that's not so it must be the declare statement if it's there
+            // as this one has to be the first statement right after <?php
+            if (preg_match('/^declare\([^)]+\);$/', $line)) {
```

Comment:
Won't this also clear a `declare(ticks=1);`, `declare(encoding='ISO-8859-1');` (and any other declare rule which might get specified in PHP 7.0+) if it should be the first statement (See [PHP.net](http://php.net/manual/de/control-structures.declare.php))?
IMO we should explicitely only capture strict_types and hence the regex should be:
```regex
/^declare\(\s*strict_types\s*=\s*[^)]+\);$/
```
Note that whitespace is allowed and therefore we need to ignore it here as well.